h2o-py/h2o/model/extensions/fairness_metrics.py

```diff
@@ -0,0 +1,35 @@
+import h2o
+from h2o.utils.typechecks import assert_is_type
+
+
+class FairnessMetrics:
+
+    def fairness_metrics(self, frame, protected_columns, reference, favorable_class):
```

Comment:
Can we add an example in the docstring?  I realize it will be a bit long... but maybe it would be helpful?